_includes/pbc/all/install-features/202304.0/marketplace/install-the-merchant-portal-feature.md

```diff
@@ -0,0 +1,391 @@
+
+
+{% info_block infoBox "Info" %}
+
+See [Marketplace Merchant Portal Core feature integration](/docs/pbc/all/merchant-management/{{page.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-merchant-portal-core-feature.html).
+
+{% endinfo_block %}
+
+
+## Environment requirements
+
+- [Node.js](https://nodejs.org/en/download/) — minimum version is v18.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm/) — minimum version is v9.
+- Spryker supported PHP version (7.3, 7.4)
```

Comment:
Should be checked by BE

Reply:
While it's feature installation for 202304.0 it should be imho Spryker supported PHP version (>=8.0) instead of Spryker supported PHP version (7.3, 7.4).